src/code/z_skelanime.c

```diff
@@ -2,12 +2,12 @@
 #include <global.h>
 #include <vt.h>
 
+#define ONE_THIRD (1.0f / 3.0f)
```

Comment:
I'm really not a fan of using a define for that, why hide `(1 / 3)` under a define that takes longer to read, write and tweak?

I could see a define being used for the specific usage of scaling the update rate, but then it should have a name that corresponds to its usage/purpose rather than its value.

Reply:
I agree, I went with actually including the division in the code when I had a similar case. I like it more personally